DOCS.md

```diff
@@ -6,6 +6,12 @@ Source Code: https://github.com/go-vela/vela-makisu
 
 Registry: https://hub.docker.com/r/target/vela-makisu
 
+{{% notice warning %}}
```

Comment:
Like the idea! But I think it would be better to keep this raw markdown. 

It does get auto-rendered into the Hugo site:
https://go-vela.github.io/docs/plugins/registry/pipeline/makisu/

But if someone is viewing it here on the repo the syntax does not render and it makes it really hard to read.

Reply:
I originally just copied I swapped the notice for it's own section of markdown - thoughts @kneal 
 @jbrockopp ? I had this change some place else that I should update once a way to format has been approved.

Reply:
I'm good with the change 👍